src/main/java/org/heigit/ohsome/ohsomeapi/executor/ElementsRequestExecutor.java

```diff
@@ -150,6 +146,19 @@ public static void extract(ElementsGeometry elemGeom, HttpServletRequest servlet
     }
   }
 
+  private static Set<Integer> keysToKeysInt(String[] keys, TagTranslator tt) {
```

Comment:
The ElementsRequestExecutor class does not have any private helper methods in it. I'd prefer if it would stay that way and you maybe put this into the helper class ExecutionUtils.java :)

I'm open though for discussion to make a bigger refactoring and structure the methods differently at some point.